### DISCUSSION TITLE
Refresh rate drops when I unplug the laptop

### DISCUSSION THREAD

**Discussion by aytacsekmen** [seerge/g-helper#3072] [Q&A]
There wasn't any issue before. But from today even if the refresh rate option in ghelper menu is set to 300hz+ OD (not the Auto option) when laptop is unplugged, it changes to 60hz when I plug and unplug it again. Asus services are all disabled from ghelper.  Any help is appreciated. 

Here is the log:
[log_ghelper.txt](https://github.com/user-attachments/files/16911390/log_ghelper.txt)

Settings:

![picture](https://github.com/user-attachments/assets/36d80810-110a-4249-9b1f-3ff6fe4b46e1)

**Comment by seerge** [comment]
@aytacsekmen hello, it's not G-Helper changing refresh rate.

Most likely you have **Asus Smart Display Control** app installed and running. [You need to stop or uninstall it](https://github.com/seerge/g-helper/wiki/Requirements#recommendations-optional) from windows

**Reply by aytacsekmen** [reply]
I previously did that step as well, but the refresh rate still changes when I unplug. Is there any way to check if it is totally uninstalled? Because even if the pop up says "Armoury crate is uninstalled.", I see the following logs under the folder C:\Users\Public\Roaming\ASUS\ASUS Smart Display Control. I'm not sure if these logs indicate the problem or not, but I wanted to share them, just in case.

[lasttimeACDC.txt](https://github.com/user-attachments/files/16911551/lasttimeACDC.txt)
[lasttimeDCRefresh.txt](https://github.com/user-attachments/files/16911552/lasttimeDCRefresh.txt)
[OD_Log.txt](https://github.com/user-attachments/files/16911553/OD_Log.txt)

**Reply by aytacsekmen** [reply]
@seerge Yeah, you were right. I checked the processes running from task manager, and stopped it running.  And now everything works fine. At the beginning I thought, when I follow the instructions you shared for the uninstallation, it won't run again. But it seems like, even if I uninstall it, it keeps running somehow. If you have any other idea to completely remove that other than armoury crate uninstallation, I will be appreciated <3

Thanks anyways.

**Reply by seerge** [reply]
@aytacsekmen well, it should be under windows Add & Remove programs.
